src/pages/Coins/Coins.styles.ts

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+
+export const Coins = styled.div`
+	margin: ${({ theme }) => theme.spacing.spacingMedium} ${({ theme }) => theme.spacing.spacingMedium};
+	padding: ${({ theme }) => theme.spacing.spacingSmall} ${({ theme }) => theme.spacing.spacingSmall};
```

Comment:
Also, if you have the same padding/margin for left & right/top & bottom, you can use a single declaration for all the 4 dimentions, like this: 'padding: ${({ theme }) => theme.spacing.spacingSmall}' and this will suffice to get the result you want